File: solutions/1319-unique-number-of-occurrences/solution.py
class Solution(object):
    def uniqueOccurrences(self, arr):
        """
        :type arr: List[int]
        :rtype: bool
        """
        ls=set(arr)
        ls1=[]
        for i in ls:
            ls1.append(arr.count(i))
        return len(ls1)==len(set(ls1))

File: solutions/1319-unique-number-of-occurrences/solution.java
class Solution {
    public boolean uniqueOccurrences(int[] arr) {
        Map<Integer,Integer>map=new HashMap<>();
        Set<Integer>set=new HashSet<>();
        int n=arr.length;
        for(int i=0;i<n;i++)
            map.put(arr[i],map.getOrDefault(arr[i],0)+1);
        for(int i : map.values())
            set.add(i);
        return map.size()==set.size();
    }
}
